tokens/tokens.css

```diff
@@ -1,9 +1,61 @@
 /**
  * Do not edit directly
- * Generated on Wed, 15 Dec 2021 14:37:30 GMT
+ * Generated on Fri, 17 Dec 2021 13:14:01 GMT
  */
 
 :root {
+  --simba-button-background-color: var(--simba-color-primary-500);
```

Comment:
These aren't useful as CSS custom props for the end user, should only do this for usable (base) tokens